src/lib/components/HeroComponent.svelte

```diff
@@ -1,6 +1,12 @@
 <script>
     import Button from './Button.svelte';
     import src from '$lib/lino-sablay.svg';
+    function scrollToContact() {
+        const contactSection = document.getElementById('contact-section');
```

Comment:
In Svelte, we discourage selecting an element by its ID. Instead, let's prefer yung [`bind:this`](https://svelte.dev/docs/element-directives#bind-this). Then let's put the browser-only logic inside an [`onMount`](https://svelte.dev/docs/svelte#onmount) function so that it doesn't fail during the build process.